Backend/nodeJS/src/modules/controllers/task-controller.js

```diff
@@ -0,0 +1,54 @@
+const shop = require('/home/user/Documents/Work/module3/Shopping list/nodeJS/src/db/models/task/Index');
+
+module.exports.getAllShops = async (req, res, next) => {
+  res.set('Access-Control-Allow-Origin', '*');
+  shop.find().then(result => {
+    res.send({ data: result });
+  });
+}
+
+module.exports.createNewShop = (req, res, next) => {
+  res.set('Access-Control-Allow-Origin', '*');
+  const body = req.body;
+  if ((body.hasOwnProperty('text') && body.hasOwnProperty('cost'))) {
+    const shops = new shop({
+      text: body.text,
+      date: body.date,
+      cost: body.cost
+    })
+    shops.save().then(result => {
+      res.send({data: result});
```

Comment:
if you use statuses, then use them everywhere:
`res.status(200).send({data: result});`